### ISSUE TITLE
Link Broken

### ISSUE THREAD

**Issue by Adonis2115** [SilentCicero/meteor-dapp-cosmo#18]
http://meteor-dapp-cosmo.meteor.com/

This link is not working

**Comment by vyorkin** [comment]
looks like the project is dead :(

**Comment by Achieve777** [comment]
yep, and the latest committing is 2 years ago

**Comment by SilentCicero** [comment]
Yeah, this project should only be used to learn from at this point. Nothing more.
